openlibrary/macros/databarView.html

```diff
@@ -27,7 +27,7 @@
     $if edit and not page.is_fake_record():
         <div class="editButton">
           <!-- FIXME: accesskey / keyboard shortcut needs i18n -->
-          <a class="linkButton larger" href="$edit_url" title="$_('Edit this template')"
+          <a class="cta-btn cta-btn--vanilla" style="margin-top: 0px;" href="$edit_url" title="$_('Edit this template')"
```

Comment:
no inline styles please

Reply:
Sure @mekarpeles. I'll remove the inline styling used here. I added it as the Edit button was looking rather misaligned.

Reply:
We can use a second css class for cases like this, though inline styled override other rules so it can cause other errors of precedence and make it hard to know which styles are winning in priority :)